lib/src/cartesian/CartesianChart.tsx

```diff
@@ -313,6 +316,12 @@ export function CartesianChart<
     ) as PointsArg;
   }, [_tData, yKeys]);
 
+  // On bounds change, emit
+  React.useEffect(() => {
+    onChartBoundsChange?.(chartBounds);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [chartBounds]);
```

Comment:
My guess is it wants `onChartBoundsChange` to be part of this.

Reply:
Yeah, but I want to allow an inline fn. I guess somewhere else we used a ref + effect to support this type of thing. Maybe we just roll our own sort of `useEvent`-like hook that can make this less ugly. I think I'll try to sneak this into this PR actually.